index.html

```diff
@@ -30,5 +30,94 @@ <h1 class="text-primary-1 ff-crete line-height-1 ">Hey There. <br> I'm Tomi</h1>
+                    <li>html</li>
+                    <li>bootstrap</li>
+                    <li>Ruby</li>
+                </ul>
+                <button class="btn btn-project" type="button"> See Project</button>
+            </div>
+            <div class="mg-1 text-light project">
+                <h2 class="mg-inline"> Profesional Art Printing Data </h2>
+                <p class="mg-1 mg-inline" > A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard </p>
+                <ul class=" mg-1 mg-inline flex tag tag-project">
+                    <li>html</li>
+                    <li>bootstrap</li>
+                    <li>Ruby</li>
+                </ul>
+                <button class="btn btn-project" type="button"> See Project</button>
+            </div>
+            <div class="mg-1 text-light project">
+                <h2 class="mg-inline"> Profesional Art Printing Data </h2>
+                <p class="mg-1 mg-inline" > A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard </p>
+                <ul class=" mg-1 mg-inline flex tag tag-project">
+                    <li>html</li>
+                    <li>bootstrap</li>
+                    <li>Ruby</li>
+                </ul>
+                <button class="btn btn-project" type="button"> See Project</button>
+            </div>
+            <div class="mg-1 text-light project">
+                <h2 class="mg-inline"> Profesional Art Printing Data </h2>
+                <p class="mg-1 mg-inline" > A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard </p>
+                <ul class=" mg-1 mg-inline flex tag tag-project">
+                    <li>html</li>
+                    <li>bootstrap</li>
+                    <li>Ruby</li>
+                </ul>
+                <button class="btn btn-project" type="button"> See Project</button>
+            </div>
+        </div>
+    </section>
+    <section class=" bg-dark container">
+        <div class="about">
+            <h2 class="text-primary-1 ff-crete ">About Me</h2>
+            <p class="mg-1 ff-inter text-light">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            <button class="btn" type="button"> Get my Resume</button>
+        </div>
+        <div class=" flex skill">
+            <h3 class="text-light ff-inai"> <span class="skill-icon"></span>   Language</h3>
+            <ul class="center ff-inter">
+                <li class="text-light mg-1">Javascript</li>
+                <li class="text-light mg-1">Ruby</li>
+                <li class="text-light mg-1">HTML</li>
+                <li class="text-light mg-1">CSS</li>
```

Comment:
- [ ] _The list of icons uses a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag)_. Please add the `a` tags and the list of icons with your GitHub and etc sites

Reply:
Thank you @GzimAsani for the review. Am I to add the link tag to official websites of the various languages and frameworks ?

Reply:
@t-bello7 Yes